src/lib/snark_profiler_lib/snark_profiler_lib.ml

```diff
@@ -435,67 +438,6 @@ let create_ledger_and_zkapps ?(min_num_updates = 1) ?(num_proof_updates = 0)
   in
   (ledger, zkapp)
 
-let _create_ledger_and_zkapps_from_generator num_transactions :
```

Comment:
This whole function was removed as unused/obsolete?

Reply:
apparently :shrug: 

It seems like we don't have 100% coverage for `warn-unused`